WooCommerce/Classes/Analytics/WooAnalyticsEvent+ProductCreationAI.swift

```diff
@@ -0,0 +1,62 @@
+import Foundation
+
+extension WooAnalyticsEvent {
+    enum ProductCreationAI {
+        private enum Key: String {
+            case value = "value"
+            case isFirstAttempt = "is_first_attempt"
+        }
+
+        static func entryPointDisplayed() -> WooAnalyticsEvent {
+            WooAnalyticsEvent(statName: .productCreationAIEntryPointDisplayed,
+                              properties: [:])
+        }
+
+        static func entryPointTapped() -> WooAnalyticsEvent {
+            WooAnalyticsEvent(statName: .productCreationAIEntryPointTapped,
+                              properties: [:])
+        }
+
+        static func productNameContinueTapped() -> WooAnalyticsEvent {
+            WooAnalyticsEvent(statName: .productCreationAIProductNameContinueTapped,
+                              properties: [:])
+        }
+
+        static func aiToneSelected(_ tone: AIToneVoice) -> WooAnalyticsEvent {
+            WooAnalyticsEvent(statName: .productCreationAIToneSelected,
+                              properties: [Key.value.rawValue: tone.rawValue])
```

Comment:
Nit: let's keep the value lowercased to follow the tracking plan. Consistent data will make it easier to analyze later.

Reply:
Great catch. 🕵️ Fixed in 739e2918a5aa9912dbaf7483c9c8c4e3b3b505d5